builtin/logical/pki/backend_test.go

```diff
@@ -1978,6 +1978,43 @@ func TestBackend_SignVerbatim(t *testing.T) {
 		t.Fatalf("sign-verbatim did not properly cap validity period on signed CSR")
 	}
 
+	// Now check signing a certificate using the not_after input using the Y10K value
+	resp, err = b.HandleRequest(context.Background(), &logical.Request{
+		Operation: logical.UpdateOperation,
+		Path:      "sign-verbatim/test",
+		Storage:   storage,
+		Data: map[string]interface{}{
+			"csr": pemCSR,
```

Comment:
One last minor nit :-)

You might want to run:

```
$ go fmt builtin/logical/pki/backend_test.go
$ go fmt builtin/logical/pki/fields.go
```

to clean up the little formatting things. Otherwise, tests pass locally and the PR looks good to me :-)

Reply:
@cipherboy I fixed the formatting, thank you for the feedback.